core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/store/filter/SpatialQueryFilter.java

```diff
@@ -196,8 +204,73 @@ public void fromBinary(
 					fieldId));
 		}
 		buf.get(theRest);
-		queryGeometry = geometryInterner.intern(GeometryUtils.geometryFromBinary(geometryBinary));
+		// cannot intern the prepared geometry. It uses a Object.hashCode().
+		preparedGeometryImage = geometryImageInterner.intern(new GeometryImage(
+				geometryBinary));
+		preparedGeometryImage.init();
 
 		super.fromBinary(theRest);
 	}
+
+	/**
+	 * Used for interning Geometry. Interning a geometry provides no additional
+	 * benefit since the geometry is already constructed in the heap at the
+	 * interning begins. The byte image of geometry provides a more efficient
+	 * component to hash and associate with a single image of the geometry.
+	 * Prepared geometries cannot be interned since they do not extend
+	 * Object.hashCode().
+	 * 
+	 * Using this approach increased performance of a large query unit test by 40% and
+	 * eliminated out of heap memory exceptions during maven builds.
```

Comment:
No.  The test ran out of heap when the code interned the geometry PRIOR to creating the prepared geometry.   The use of the prepared geometry required an alternate approach to interning the Geometry.  I am not sure what you mean by not being benchmarked.  This was a separate effort from the benchmark work I did yesterday.